packages/integration/src/github/GithubCredentialsProvider.ts

```diff
@@ -121,22 +120,15 @@ class GithubAppManager {
     });
   }
 
+  getInstallations(): Promise<
+    RestEndpointMethodTypes['apps']['listInstallations']['response']['data']
+  > {
+    return this.appClient.paginate(this.appClient.apps.listInstallations);
```

Comment:
Wasn't sure the best way to handle etags here for a paginated request. Currently this looks like it only fetched the first 30 installations according to the [docs](https://docs.github.com/en/rest/reference/apps#list-installations-for-the-authenticated-app). I updated this to use `paginate` to ensure this scales to fetch all installations but this doesn't give us an etag for us to track